packaging/kryoptic.spec

```diff
@@ -38,27 +39,29 @@ A PKCS #11 software token written in Rust.}
 %description %{_description}
 
 %package tools
-Requires:       %{name} = %{version}-%{release}
-Summary:        Supporting tools for kryoptic software token.
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+Summary:        Supporting tools for kryoptic software token
 
 %description tools
-Supporting tools for kryoptic software token. Most notably a migration tool for the SoftHSM database.
+Supporting tools for kryoptic software token.
+Most notably a migration tool for the SoftHSM database.
 
 %prep
-%autosetup -n kryoptic-%{revision} -p1
+%autosetup -n kryoptic-%{commit} -p1
 %cargo_prep
 
 %generate_buildrequires
 %cargo_generate_buildrequires -f dynamic,nssdb,standard
 
 %build
-CONFDIR=%{_sysconfdir} %cargo_build -f dynamic,nssdb,standard
+export CONFDIR=%{_sysconfdir}
+%cargo_build -f dynamic,nssdb,standard
 %{cargo_license_summary -f dynamic,nssdb,standard}
 %{cargo_license -f dynamic,nssdb,standard} > LICENSE.dependencies
 
 %install
 %cargo_install -f dynamic,nssdb,standard
-install -Dp target/rpm/libkryoptic_pkcs11.so $RPM_BUILD_ROOT/%{_libdir}/libkryoptic_pkcs11.so
+install -Dp target/rpm/libkryoptic_pkcs11.so $RPM_BUILD_ROOT/%{_libdir}/pkcs11/libkryoptic_pkcs11.so
```

Comment:
shouldn't the dir be something like %{_libdir}/kryoptic/libkryoptic_pkcs11.so ?

Reply:
The PKCS#11 packaging guidelines say that the directory should be `pkcs11`. It makes it easier at least for the p11-kit, but also for discoverability:
https://docs.fedoraproject.org/en-US/packaging-guidelines/Pkcs11Support/
We could also consider shipping the `p11-kit` module to load kryoptic automatically, which might be also useful, but I was too lazy to add it so far. I can add it to this PR too